### PR TITLE
docs($rootScope.Scope): grammar

### PR DESCRIPTION
**Pull request by thorn0**

### CODE REVIEW

File: src/ng/rootScope.js [hunk 1]
```diff
@@ -437,7 +437,7 @@ function $RootScopeProvider() {
        * If any one expression in the collection changes the `listener` is executed.
        *
        * - The items in the `watchExpressions` array are observed via standard $watch operation and are examined on every
-       *   call to $digest() to see if any items changes.
+       *   call to $digest() to see if any items change.
```
Comment (gkalpak) [inline src/ng/rootScope.js]: This makes it sound as if the act of examining itself may cause the change. I think it is more clear to say: _"if any item(s) changed"_

Comment (thorn0) [inline src/ng/rootScope.js]: What if we put it this way?

*The items in the `watchExpressions` array are observed via standard `$watch` operation. Their return values are examined for changes on every call to `$digest`.*

Comment (gkalpak) [inline src/ng/rootScope.js]: Works for me. I'll change it to what you suggested while merging.

**Comment by gkalpak** [comment]
@googlebot, could you please do your CLA thing?

**Comment by gkalpak** [comment]
@thorn0, could you comment on the PR. Maybe that would trigger @googlebot.

**Comment by thorn0** [comment]
What's wrong with @googlebot?

**Pull request closed without merging**